packages/pwa-kit-create-app/assets/bootstrap/js/overrides/app/routes.jsx.hbs

```diff
@@ -18,20 +24,60 @@ const fallback = <Skeleton height="75vh" width="100%" />
 
 // Create your pages here and add them to the routes array
 // Use loadable to split code into smaller js chunks
+{{#if answers.project.hybrid}}
+const Home = loadable(() => import('@salesforce/retail-react-app/app/pages/home'), { fallback })
+{{else}}
 const Home = loadable(() => import('./pages/home'), {fallback})
+{{/if}}
 const MyNewRoute = loadable(() => import('./pages/my-new-route'))
+{{#if answers.project.hybrid}}
+const PageNotFound = loadable(() => import('@salesforce/retail-react-app/app/pages/page-not-found'))
+{{/if}}
 
 const routes = [
     {
         path: '/',
         component: Home,
         exact: true
     },
+    {{#if answers.project.hybrid}}
```

Comment:
Why do we want `/home` in hybrid projects, but not regular ones?

Reply:
/home is added as there are cases where SFRA tries to route to the /home controller. We want PWA to handle those cases